src/act.wizard.cpp

```diff
@@ -2535,11 +2535,11 @@ ACMD(do_award)
   else
     gain_karma(vict, k, TRUE, FALSE, FALSE);
 
-  send_to_char(vict, "You have been awarded %0.2f karma for %s.\r\n", (float)k*0.01, reason);
+  send_to_char(vict, "You have been awarded %0.2f karma for %s\r\n", (float)k*0.01, reason);
 
-  send_to_char(ch, "You awarded %0.2f karma to %s for %s.\r\n", (float)k*0.01, GET_CHAR_NAME(vict), reason);
+  send_to_char(ch, "You awarded %0.2f karma to %s for %s\r\n", (float)k*0.01, GET_CHAR_NAME(vict), reason);
```

Comment:
These two send_to_chars can also accept the inline punctuation checker. Something like `"You awarded %0.2f karma to %s for %s%s\r\n", [...], ispunct(reason[strlen(reason) - 2]) ? "" : ".");`